.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @Sage-Bionetworks/fair-data
+* @Sage-Bionetworks/python-client-developers
```

Comment:
If you could provide this team write access to this repo, that would be fantastic!

Reply:
there's no schematic-developer team. I added @Sage-Bionetworks/python-client-developers manually here: 
<img width="825" alt="Screenshot 2024-08-25 at 5 04 18 PM" src="https://github.com/user-attachments/assets/c4dcd9e4-2e99-4f7f-8380-096b51370d1f">

Reply:
just changed the role to "maintain"